### ISSUE TITLE
links to some labs not working & markdown for student handouts missing.

### ISSUE THREAD

**Issue by ccrocke1** [integrated-ethics/web#50]
Hi Randy, I noticed a couple of the links in the ie: Labs Overview are not working. ( Sorry if you are already aware & I'm just repeating).

* [x] Link for student handout Markdown version gives a 404: Not Found message for the following labs-
Why Ethics?
A first look...
Utilitarianism
Analogies
Hospitality
Avoiding Algorithmic Bias- long version & scenarios (EU AI Guidelines- is fine)
Open Source Accessibility - worksheet only

* [x] Link for student materials- webpage goes to a "We're sorry..."
Utilitarianism- sorry I didn't catch his earlier. 
* [x] Media Literacy lab- not linked get a "We're sorry.."
* [x] Same is true for Data Cleaning- but I think that's still an open issue.

**Comment by rpruim** [comment]
for utilitarianism: the link was correct, but the actual filename had a typo.  I've fixed.

for medial literacy and data cleaning, my regex global substitution choked on the extra parens in the title.  I've fixed.  (This should be a one time issue introduced by my change to using a shortcode to generate these titles and links.)

For the markdown links, I might write a shortcode so that this doesn't rely so heavily an getting the file extension correct.  (The file extension sometimes gets adjusted based on features of the lab; then one needs to remember to update the github link to match.  I'd like to automate that instead.). Stay tuned.
